modules/microshift-firewall-known-issue.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-firewall-known-issue_{context}"]
+== Known firewall issue
+To avoid breaking traffic flows with a firewall restart, execute firewall commands before starting OVN-Kubernetes pods. OVN-Kubernetes makes use of iptable rules for some traffic flows, such as those using the NodePort service. The iptable rules are generated and inserted by the `ovnkube-master` container, but are deleted when the firewall restarts. The absence of the iptable rules breaks traffic flows. If firewall commands have to be executed after OVN-Kubernetes pods have started, manually restart the `ovnkube-master` pod to trigger a reconciliation of the iptable rules.
```

Comment:
I think instead of "OVN-Kubernetes pods" we just want to say "MicroShift". The user doesn't have any way of doing anything between the time they start MicroShift and when those pods start up. There's also a level of implementation detail that most users should not need and that we don't want to have to keep up to date over time, so I've suggested some wording changes to keep this content evergreen. We especially don't want users killing individual pods, since we want them to treat MicroShift as a unit.

```suggestion
To avoid breaking traffic flows with a firewall restart, execute firewall commands before starting MicroShift. The CSI driver in MicroShift makes use of iptable rules for some traffic flows, such as those using the NodePort service. The iptable rules are generated and inserted by the CSI driver, but are deleted when the firewall restarts. The absence of the iptable rules breaks traffic flows. If firewall commands have to be executed after MicroShift has started, MicroShift needs to be restarted to reset the rules controlled by the CSI driver.
```